handlers.c

```diff
@@ -1205,9 +1201,7 @@ bool handler__watch(globals_t * vars, char **argv, unsigned argc)
             valcpy(&o, &n);
             truncval(&o, &old_val);
 
-            if (valtostr(&o, buf, sizeof(buf)) == false) {
-                strncpy(buf, "unknown", sizeof(buf));
-            }
+            valtostr(&o, buf, sizeof(buf));
```

Comment:
These two are only relevant to the 'watch' command which is not used by the GUI.